File: CS315-HW2/Codes/altiparmak_gokberk.py
# Conditional exit
for x in range(3):
    print("*")
print("\n")

# Unconditional exit(break)
x = 3
while True:
    print("***\n")
    if (x == 3):
        break

# Unconditional exit(continue)
while True:
    x = x + 1
    if (x < 10):
        continue
    print("******\n")
    break

# Enclosed loops exit
for i in range(3):
    for j in range(2):
        if (i == 1):
            break
        print("j =",j)
    print ("i =" ,i)

File: CS315-HW2/Codes/altiparmak_gokberk.js
// Conditional exit
for(let i = 0; i < 5; i++) {
    console.log("*");
}

// Unconditional exit(break)
var x = 4;
for (;;) {
    if (x == 5)
        break;
    console.log("*");
    x++;
}

// Unconditional exit(continue)
for (;;) {
    x++;
    if (x < 10)
        continue;
    console.log("*");
    break;
}

// Enclosed loops exit
for (let i = 0; i < 3; i++) {
    for (let j = 0; j < 2; j++) {
        if (i == 1) {
            break;
        }
        console.log("j = " + j);
    }
    console.log("i = " + i);
}